test/versioned/hapi/hapi-v20/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "hapi-v17-tests",
+  "name": "hapi-v20-tests",
```

Comment:
Looks like we've dropped hapi-post18 with hapi-v20 as the replacement for all tests. It is interesting that we've ported over some of the tests from 17 which were defined explicitly in post-18, which was the previous coverage for 20. Unsure if there were any differences there to be aware of or not in those test files.

Reply:
> With this change, we fully don't support the `hapi` module that is not namespaced.
> 
> As such, we should remove from `'hapi': { type: MODULE_TYPE.WEB_FRAMEWORK },` from instrumentations and can cleanup anything related to that. 

I've removed that from instrumentations. The `hapi` and `@hapi/hapi` instrumentation structure is pretty DRY: the namespaced one points back at the other.  I could refactor that in this PR to consolidate it all under the namespaced directory.